packages/composables/src/composables/useFloating.ts

```diff
@@ -304,7 +310,9 @@ export const useFloating = (
 
       rect.value = safeElementDomRef.value.getBoundingClientRect()
 
-      args.callback && args.callback(rect, openerRect, containerRect, position, actualPosition)
+      args.callback &&
+        args.callback.value &&
+        args.callback.value(rect, openerRect, containerRect, position, actualPosition)
```

Comment:
I suggest avoiding logical operations in place of if statements as they make code less clear
```suggestion
      if (args.callback?.value) {
        args.callback.value(rect, openerRect, containerRect, position, actualPosition)
      } 
```